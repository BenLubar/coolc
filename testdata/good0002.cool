class Main() extends IO() {
	{
		out("2 + 3 = ").out_any(2 + 3).out("\n");
		out("2 - 3 = ").out_any(2 - 3).out("\n");
		out("2 * 3 = ").out_any(2 * 3).out("\n");
		out("2 / 3 = ").out_any(2 / 3).out("\n");
		out("3 + 2 = ").out_any(3 + 2).out("\n");
		out("3 - 2 = ").out_any(3 - 2).out("\n");
		out("3 * 2 = ").out_any(3 * 2).out("\n");
		out("3 / 2 = ").out_any(3 / 2).out("\n")
	};
}
