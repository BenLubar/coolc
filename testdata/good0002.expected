2 + 3 = 5
2 - 3 = -1
2 * 3 = 6
2 / 3 = 0
3 + 2 = 5
3 - 2 = 1
3 * 2 = 6
3 / 2 = 1
